api/provision/start-provision-job.test.ts

```diff
@@ -0,0 +1,120 @@
+import { Context } from "aws-lambda";
+import { handler } from "./start-provision-job";
+import { CloudServiceProvider, Network } from "../../models/cloud-service-providers";
+import { ProvisionRequestType } from "../../models/provisioning-jobs";
+import { ApiSuccessResponse, ValidationErrorResponse } from "../../utils/response";
+import { sfnClient } from "../../utils/aws-sdk/step-functions";
+import { mockClient } from "aws-sdk-client-mock";
+
+const sfnMock = mockClient(sfnClient);
+beforeEach(() => {
+  sfnMock.reset();
+});
+
+const fundingSources = [
+  {
+    taskOrderNumber: "1234567890123",
+    clin: "9999",
+    popStartDate: "2021-07-01",
+    popEndDate: "2022-07-01",
+  },
+];
+const operators = ["admin1@mail.mil", "superAdmin@mail.mil"];
+const provisioningBodyNoPayload = {
+  jobId: "81b31a89-e3e5-46ee-acfe-75436bd14577",
+  userId: "21d18790-bf3e-4529-a361-460ee6d16e0b",
+  portfolioId: "b02e77d1-234d-4e3d-bc85-b57ca5a93952",
+  operationType: ProvisionRequestType.ADD_OPERATORS,
+  targetCsp: CloudServiceProvider.CSP_A.name,
+  targetNetwork: Network.NETWORK_1,
+};
+
+describe("Successful provisioning operations", () => {
+  it("should add a new portfolio", async () => {
+    const request = {
+      body: {
+        ...provisioningBodyNoPayload,
+        payload: {
+          name: "Sample Portfolio",
+          fundingSources,
+          operators,
+        },
+      },
+      requestContext: { identity: { sourceIp: "9.9.9.9" } },
```

Comment:
Should we bother to extract this hypothetical value and re-use?

Reply:
Yeah, that works. It didn't cross my mind.

Reply:
Small nitpick here if you happen to be in the file again: If we don't care what the `sourceIp` really is, can we pick an address from [the RFC 5737 list](https://datatracker.ietf.org/doc/html/rfc5737#section-3)? `9.9.9.9` is a semi-common DNS service provider so picking something from one of the TEST-NET-N ranges might make it a little more clear that the value is meaningless.